### ISSUE TITLE
Displays random subtitle lines

### ISSUE THREAD

**Issue by LinAGKar** [jellyfin/jellyfin-chromecast#576]
**Describe the bug**

I'm playing an MKV file with a large number of embedded SRT subtitles in different languages, with the first track being in English. If I play with English subtitles, ~it plays them fine~ (I've gotten it once with English subtitles, but it usually doesn't happen). However, if I choose a different language, it displays subtitle lines at random, taking lines from random languages (choosing different languages for each line), and from random timestamps. Sometimes it even shows multiple languages on top of eachother or just displays an empty (very narrow) subtitle box. This seems to have started happening around the time I updated to Jellyfin 10.9, but I'm only seeing this behavior on Chromecast, not on web or Android.

**System (please complete the following information):**

-   OS: Docker
-   Browser: Android Client
-   Jellyfin version: 10.9.2
-   Cast Receiver version: Stable
-   Cast client: 2nd gen

**Comment by dpad** [comment]
For reference, some others are seeing this exact same behaviour. Probably related to https://github.com/jellyfin/jellyfin/issues/12113

**Comment by 3flex** [comment]
Should be fixed in Jellyfin 10.10.5 by https://github.com/jellyfin/jellyfin/pull/13384

**Comment by LinAGKar** [comment]
Yep, this seems to work better after updating to latest Jellyfin.
